test/FormGroupSpec.js

```diff
@@ -46,6 +46,15 @@ describe('FormGroup', function() {
     assert.ok(ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'form-group-lg'));
   });
 
+  // This test case must come first, since the error only gets logged once.
```

Comment:
I agree. It was annoying enough that I was thinking of deleting this test case for a bit. "Doesn't throw a warning" isn't really an interesting or useful test in most cases.